channels/applicationinsights-channel-js/src/Sender.ts

```diff
@@ -1266,6 +1394,7 @@ export class Sender extends BaseTelemetryPlugin implements IChannelControls {
 
     /**
      * xhr state changes
+     * @deprecated
```

Comment:
when marked as deprecated, is it safe to remove `self._xhrReadyStateChange` and put everything inside  `self._xhrReadyStateChange` to a private function?

Reply:
don't need to remove the function. will remove it when we completely disable the function.